back/test/data/location.py

```diff
@@ -35,11 +36,37 @@ def another_location_data():
     return data
 
 
+def null_box_state_location_data():
+    data = default_location_data().copy()
```

Comment:
Copying is redundant because the `data()` function returns a new dictionary at every call.